selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/lane_change_settings.cc

```diff
@@ -0,0 +1,174 @@
+//
+// Created by kumar on 3/10/25.
+//
+/**
+ * Copyright (c) 2021-, Haibin Wen, sunnypilot, and a number of other contributors.
+ *
+ * This file is part of sunnypilot and is licensed under the MIT License.
+ * See the LICENSE.md file in the root directory for more details.
+ */
+
+#include "selfdrive/ui/sunnypilot/qt/offroad/settings/lateral/lane_change_settings.h"
+
+#include <map>
+#include <string>
+#include <tuple>
+#include <vector>
+
+LaneChangeSettings::LaneChangeSettings(QWidget* parent) : QWidget(parent) {
+  QVBoxLayout* main_layout = new QVBoxLayout(this);
+  main_layout->setContentsMargins(50, 20, 50, 20);
+  main_layout->setSpacing(20);
+
+  // Back button
+  PanelBackButton* back = new PanelBackButton(tr("Back"));
+  connect(back, &QPushButton::clicked, [=]() { emit backPress(); });
+  main_layout->addWidget(back, 0, Qt::AlignLeft);
+
+  ListWidgetSP *list = new ListWidgetSP(this, false);
+  // param, title, desc, icon
+  std::vector<std::tuple<QString, QString, QString, QString>> toggle_defs{
+      /***
+    {
+      "BelowSpeedPause",
+      tr("Pause Lateral Below Speed with Blinker"),
+      tr("Enable this toggle to pause lateral actuation with blinker when traveling below the desired speed selected below."),
+      "../assets/offroad/icon_blank.png",
+    },
+    ***/
+    {
+      "AutoLaneChangeBsmDelay",
+      tr("Auto Lane Change: Delay with Blind Spot"),
+      tr("Toggle to enable a delay timer for seamless lane changes when blind spot monitoring (BSM) detects a obstructing vehicle, ensuring safe maneuvering."),
+      "../assets/offroad/icon_blank.png",
+    },
+      /***
+    {
+      "RoadEdge",
+      tr("Block Lane Change: Road Edge Detection"),
+      tr("Enable this toggle to block lane change when road edge is detected on the stalk actuated side."),
+      "../assets/offroad/icon_blank.png",
+    }
+    ***/
+  };
+
+  // Controls: Auto Lane Change Timer
+  auto_lane_change_timer = new AutoLaneChangeTimer();
+  auto_lane_change_timer->setUpdateOtherToggles(true);
+  auto_lane_change_timer->showDescription();
+  connect(auto_lane_change_timer, &OptionControlSP::updateLabels, auto_lane_change_timer, &AutoLaneChangeTimer::refresh);
+  connect(auto_lane_change_timer, &AutoLaneChangeTimer::updateOtherToggles, this, &LaneChangeSettings::updateToggles);
+  list->addItem(auto_lane_change_timer);
+
+  // Pause Lateral Below Speed w/ Blinker
+  /***
+  pause_lateral_speed = new PauseLateralSpeed();
+  pause_lateral_speed->showDescription();
+  connect(pause_lateral_speed, &OptionControlSP::updateLabels, pause_lateral_speed, &PauseLateralSpeed::refresh);
+
+  for (auto &[param, title, desc, icon] : toggle_defs) {
+    auto toggle = new ParamControlSP(param, title, desc, icon, this);
+
+    list->addItem(toggle);
+    toggles[param.toStdString()] = toggle;
+
+    if (param == "BelowSpeedPause") {
+      list->addItem(pause_lateral_speed);
+    }
+  }
```

Comment:
The UI crashes in `updateToggles()` because `toggles["AutoLaneChangeBsmDelay"]` is never initialized - the loop that populates the `toggles` map is currently commented out.

To fix this, uncomment the following block to ensure `ParamControlSP` instances are created and stored in `toggles`.

Reply:
done